README.md

```diff
@@ -1,32 +1,297 @@
-# Creating Objects
+# Creating A Class Instance
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Create an object in Java using the `new` operator
+- Initialize a field based on data type
+- Declare a variable having a class type
+- Assign a variable to reference an object
+- Initialize a field at variable declaration
 
-## Introduction
+## Instantiating An Object
 
-A 1-2 sentence summary of what will be covered.
+As stated before, in Java, an **object** is an _instance_ of a **class**.
+The  class serves as a blueprint describing the data structure for each object,
+along with the methods that implement the object behavior.
 
-## Topic 1
+Let's go back to our `Dog` class from earlier:
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+```java
+public class Dog {
+
+  String name;
+  String breed;
+  int age;
+  boolean waggingTail;
+
+}
+```
+
+How do we go about creating an instance of `Dog` in Java?
+
+When we create an instance of this class, we are telling Java to create something
+that will comply with the data structure described by the class. In other
+words, each dog object needs memory to be allocated to store values for the
+`name`, `breed`, `age`, and `waggingTail` properties.
+
+If we want to create an instance of `Dog`, we can do so with the following code snippet:
+
+```java
+new Dog()
+```
+
+The code consists of 2 parts:
+
+- Instantiation − The `new` operator is used to create an instance of `Dog` by allocating memory
+  to store the four instance variables defined in the `Dog` class (name, breed, age, waggingTail).
+
+- Initialization − The `new` operator is followed by a call to a constructor `Dog()`.  The
+  constructor is a method that initializes the new object by assigning a value to each instance variable.
+  We will cover constructors in detail in a later lesson.  But for now just understand that
+  the default constructor initializes each variable based on their data type:   
```

Comment:
I like this! This shows the differences between these two words, that are sometimes confused or used to mean the other.